lib/rubocop/config.rb

```diff
@@ -18,6 +18,7 @@ class Config
     # 2.1 is the oldest officially supported Ruby version.
     DEFAULT_RUBY_VERSION = 2.1
     KNOWN_RUBIES = [1.9, 2.0, 2.1, 2.2, 2.3, 2.4].freeze
+    DEFAULT_RAILS_MINOR_VERSION = 5.0
```

Comment:
I think this should be `DEFAULT_RAILS_VERSION`, because to be picky, the minor version here is just `0`. 😉